### ISSUE TITLE
XML 1.1

### ISSUE THREAD

**Issue by hober** [w3ctag/obsoletion#6]
Hello TAG!

@murata2makoto [requests](https://www.w3.org/mid/CALvn5EDwPnNQnYkrcRvW4rpaxfNPSbzy6on894B7v=9ixOM7-Q@mail.gmail.com) that the TAG consider the following specification:
  - [XML 1.1](https://www.w3.org/TR/xml11/)

be considered by the TAG for (choose one; see [§6.9 Obsoleting or Rescinding a W3C Recommendation](https://www.w3.org/Consortium/Process/#rec-rescind) of the Process for what this means):
  - [x] obsoleting (of a W3C Recommendation)
  - [ ] superseding (of a W3C Recommendation)
  - [ ] rescinding (of a W3C Recommendation)
  - [ ] restoring (of an Obsolete W3C Recommendation)

No rationale was provided.

This specification was produced by the [XML Core WG](https://www.w3.org/XML/Core/) which was closed in 2016.

The following specifications have dependencies on this specification:
 - list not provided

The following implementations of this specification are known:
 - list not provided

**Comment by murata2makoto** [comment]
XML 1.0 Fifth Edition obsoletes XML 1.1.

**Comment by dbaron** [comment]
Are there data on the amount of adoption of both (a) XML 1.1 and (b) the changes in the fifth edition of XML 1.0?  (For the latter, probably relative to all XML 1.0 use seems relevant.)

**Comment by murata2makoto** [comment]
I do not have any data.  But I know that W3C created the fifth edition of XML 1.0 because W3C believed that XML 1.1 will not take off.  Microsoft does not support XML 1.1.

**Comment by ylafon** [comment]
IIRC, there is still a difference with NEL being part of line-end characters in 1.1 and not in 1.0, right?

**Comment by murata2makoto** [comment]
Right.  IBM cared at that time.

**Comment by ndw** [comment]
As @murata2makoto notes, the significant feature of XML 1.1, expanding the repertoire of Unicode characters was incorporated into 1.0 Fifth Edition (largely because 1.1 failed to achieve the adoption one might have hoped at the time).

The other features: NEL as an end-of-line character and the ability to use &#x1-&#x1f as numeric character references (and the unfortunate decision to require &#x7f-&#x9f to be numeric character references) are probably what doomed it.

I doubt that there's any serious adoption of 1.1 outside of test suites. 

I don't understand how obsoleting it will improve things, and I'm ambivalent about doing it, but if "minimal usage" is a motivational factor for obsoleting a recommendation, I expect it's fair to say that XML 1.1 has minimal usage.

**Comment by murata2makoto** [comment]
If W3C recommends something, it is difficult for people to try something different.  Withdrawing unused recommendations allow people to try what they want.

**Comment by timbray** [comment]
Should someone be so foolish as to start providing or transmitting data encoded in XML1.1, it's unlikely that it would be received and processed successfully.  So obsoleting it feels to me like a service to the community.  Are there any other W3C mechanisms for hanging a "Do Not Use This" sign on a recommendation?

**Comment by skynavga** [comment]
This will impact other W3C RECs that make normative reference to XML1.1, such as TTML which has normatively referenced XML1.1 since first published [1] in 2010. Indeed, it is so referenced in all three editions of TTML1 and in the current (1st) edition of TTML2 [2] as well as the upcoming 2nd edition of TTML2 [3]. Note that TTML has been widely deployed and is itself normatively referenced by more than a few national and industry standards/specifications.

[1] https://www.w3.org/TR/2010/REC-ttaf1-dfxp-20101118/
[2] https://www.w3.org/TR/2018/REC-ttml2-20181108/
[3] https://w3c.github.io/ttml2/index.html

**Comment by murata2makoto** [comment]
@skynavga TTML uses the production [4a] of XML 1.1 for defining permissible names, but it does not use XML 1.1 documents.  Am I correct?  If so,  copying that production from XML 1.1 will solve the problem.

**Comment by skynavga** [comment]
@murata2makoto yes, that is correct, and it is possible to update the production in a new (4th) edition of TTML1, which is not yet planned at this time, as well as in the 2nd edition of TTML2 (under development);

**Comment by dbaron** [comment]
It sounds like there's a good bit of consensus that XML 1.1 was the wrong approach.  Are you aware of people who would argue today that XML 1.1 is still the right path?

------

For what it's worth, the reason I was asking about how widely adopted the changes in the fifth edition of XML 1.0 are is that if they're widely adopted today, that suggests that desire to use those changes wouldn't lead to future adoption of XML 1.1.

Whereas if they weren't yet widely adopted, it would seem like the market hasn't yet decided which path it's going to take to expand the repertoire of Unicode characters.  (And even if we think it will go one way, that doesn't prove it.)

**Comment by palemieux** [comment]
Assuming that there is consensus within the W3C to obsolete XML 1.1, i.e. that XML 1.1 should no longer be used, the roadmap to obsoletion should include a substantial period for review and comments, e.g. 6 months, by external organizations with whom W3C has a liaison or are known users of XML, including ETSI, SMPTE, ISO/IEC, IETF, ITU, DVB, EBU...

A quick Google search indicates that each of these organizations has specifications that reference XML 1.1.

**Comment by mendelsohn** [comment]
I am not sufficiently involved in these matters any more to offer an informed opinion the merits of making XML 1.1 obsolute as proposed, but I wanted to mention that XML Schemas 1.1 is among the specifications that makes quite extensive reference to XML 1.1. Since there is, as far as I know, no active XML Schema Working Group there might be some question as to who in practice would be able to work through any revisions needed to remove dependencies on XML 1.1. I doubt there's anything deeply hard about doing the removal once the specifications are thoroughly understood and thought through, but my impression is that XML 1.1 dependencies affect numerous details in both Schema Structures and Schema datatypes. As I recall (I haven't looked at all this for years) the dependencies are both direct and also indirect via dependencies on Namespaces 1.1 (which in turn references XML 1.1 as I recall).  My intuition is that it's important that we emerge with a schema recommendation covering today's common practice of, I presume, writing schema documents using XML 1.0 Fifth Edition to validate using documents written in XML 1.0 5th edition.

**Comment by murata2makoto** [comment]
As I see it, XML Schemas 1.1 does not have any problems even if XML 1.1 is withdrawn.  See
["1.4 Dependencies on Other Specifications"](https://www.w3.org/TR/xmlschema11-1/#intro-relatedWork) in XSD 1.1.

**Comment by hober** [comment]
We're way past the 90 day deadline on this one. Can this be closed?

**Comment by phax** [comment]
Out of curiosity: this issue is lying dormant for basically 6 years now and even TTML2 updated it's references to use XML 1.0 instead. So maybe it is time to send XML 1.1 on the deprecation track now...???

**Comment by palemieux** [comment]
Were liaisons ever shared with external organizations with whom W3C has a relationship?

**Comment by timbray** [comment]
Can we please do this? 1.1 is an annoying distraction that keeps coming up in loosely-related standards discussions that want to "reference the latest XML" and I'm tired of explaining to people who don't know the history (and shouldn't have to) why that should be 1.0 5ed rather than 1.1.

**Comment by ylafon** [comment]
Sorry it took more time than anticipated. Closing the issue here and continuing the [obsoletion process](https://www.w3.org/policies/process/20250818/#deactivation)

**Comment by phax** [comment]
This is good news :)

**Comment by timbray** [comment]
If there's any way I can help with the marking-obsolete process, I'd be happy to.
